.drone.yml

```diff
@@ -19,7 +19,7 @@ steps:
   pull: always
   image: golang:1.11-alpine
   commands:
-  - apk --no-cache add -U python3 git
+  - apk --no-cache add -U python3 git python3-dev gcc libc-dev libffi-dev openssl-dev
```

Comment:
These changes don't seem to be related to this PR, just like changes below in Dockerfiles

Reply:
@lafriks You can't build the project without these changes currently.

Reply:
@lafriks @tboerger Please advise